Documentation/Testing/ExtensionTesting.rst

```diff
@@ -2,17 +2,24 @@
 .. index:: pair: Testing; Extensions
 .. _testing-extensions:
 
+
+
 =================
 Extension testing
 =================
 
-..  attention::
-    The example in this chapter covers only outdated TYPO3 and PHP versions. Help us bring
-    it up to date: See :ref:`h2document:contribute`.
+.. attention::
+
+    This page has not been fully updated yet. Specifically the following:
 
-    The general principles demonstrated here still apply.
+    *   Decide if multi-core-version testing should be documented or testing only
+        one core version. (Originally, this page was written for testing only one
+        core version since testing 2 core versions was not supported well at the
+        time. Also, when this page was originally written, enetcache tested only one core
+        version and supported only one core version per branch).
 
-..  todo: Update the examples bellow
+    For now, we document testing only one core version here, even if 2 is possible
```

Comment:
So, not only in "TYPO3 Core", but also in "Core"? Find that a bit weird, but will change.

I am removing the note at the top anyway. It is awkward and the information should have been incorporated into "Introduction" and "Scope". Since I now have 3 separate PRs for three versions, I would like to ideally finish this with as little additional commits (which have to be individually applied to all three versions) as possible and make more changes in separate PR where they can be easily backported. Will address changes in review though.